apps/shopify/shopify.star

```diff
@@ -0,0 +1,554 @@
+            display = "America/Chicago",
+            value = "America/Chicago",
+        ),
+        schema.Option(
+            display = "America/Denver",
+            value = "America/Denver",
+        ),
+        schema.Option(
+            display = "America/Phoenix",
+            value = "America/Phoenix",
+        ),
+        schema.Option(
+            display = "America/Los_Angeles",
+            value = "America/Los_Angeles",
+        ),
+        schema.Option(
+            display = "America/Anchorage",
+            value = "America/Anchorage",
+        ),
+        schema.Option(
+            display = "Pacific/Honolulu",
+            value = "Pacific/Honolulu",
+        ),
+    ]
+    metric_options = [
+        schema.Option(
+            display = "Revenue",
+            value = "revenue",
+        ),
+        schema.Option(
+            display = "Orders",
+            value = "orders",
+        ),
+        schema.Option(
+            display = "Units",
+            value = "units",
+        ),
+    ]
+
+    alignment_options = [
+        schema.Option(
+            display = "Left",
+            value = "start",
+        ),
+        schema.Option(
+            display = "Center",
+            value = "center",
+        ),
+        schema.Option(
+            display = "Right",
+            value = "end",
+        ),
+    ]
+
+    font_options = [
+        schema.Option(
+            display = "tb-8",
+            value = "tb-8",
+        ),
+        schema.Option(
+            display = "Dina_r400-6",
+            value = "Dina_r400-6",
+        ),
+        schema.Option(
+            display = "5x8",
+            value = "5x8",
+        ),
+        schema.Option(
+            display = "6x13",
+            value = "6x13",
+        ),
+        schema.Option(
+            display = "10x20",
+            value = "10x20",
+        ),
+        schema.Option(
+            display = "tom-thumb",
+            value = "tom-thumb",
+        ),
+        schema.Option(
+            display = "CG-pixel-3x5-mono",
+            value = "CG-pixel-3x5-mono",
+        ),
+        schema.Option(
+            display = "CG-pixel-4x5-mono",
+            value = "CG-pixel-4x5-mono",
+        ),
+    ]
+
+    return schema.Schema(
+        version = "1",
+        fields = [
+            schema.Text(
+                id = "store_name",
+                name = "Shopify Store Name",
+                desc = "The Shopify store name used for API access",
+                icon = "store",
+            ),
+            schema.Text(
+                id = "api_token",
+                name = "API Token",
+                desc = "The API Token for your Shopify Private App",
+                icon = "key",
+            ),
+            schema.Text(
+                id = "display_name",
+                name = "Company Name",
+                desc = "The company name to display",
+                icon = "signHanging",
+            ),
+            schema.Dropdown(
```

Comment:
A simpler way of allowing a custom timezone to be selected is to use `schema.Location`. The location will default to the users device location, but allows the user to select some other location if so desired. The location object itself includes a timezone field.

Another approach is to drop timezone entirely, and use the magic `$tz` variable that's passed by the Tidbyt runtime environment. It's automatically set to the users device location's timezone. You can find examples by searching for `$tz` in this repo.

Reply:
I chose to do this because I have seen some businesses that have financial source of truth in different than timezones than the person who might be located with a Tidbyt (my company included, I work CT but the company financials are based in ET).

"Location" fills a bit weird here to me.

Reply:
Yeah, I agree that Location is a bit weird for this use case, but while listing timezones in a dropdown works OK for users in the US, it gets really messy when we take the rest of the world into consideration. =\

Reply:
Is there plans to support a Schema for Timezones explicitly? Probably wouldn't be hard to just have the whole list.

Reply:
No immediate plan. If it was added, I suspect the best UX would still be to base it on location rather than on a full list. There're so many timezone designations out there that it'd have to be searchable, and location is a pretty convenient way to implement that search. Especially for non-technical users who've never seen a timezone designation before.

Reply:
I'll update to support the Location schema.